### ISSUE TITLE
Add new ispc

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/compiler-explorer#1253]
New version of ispc, plus flag for intel syntax: https://groups.google.com/forum/#!topic/ispc-users/zPz_1ppC-dQ

**Comment by pbrubaker** [comment]
Hey Matt, we're coming up on a new release of ISPC and I'd really love to get this addressed.  The new version of ISPC will probably drop in the next week or so.  Myself and Jon Kennedy (also Intel) recently plugged CE in our ISPC 2019 GDC talk, so I'd love to get both versions up.

**Comment by mattgodbolt** [comment]
The intel support and 1.10.0 are now live. Once the new compiler is ready let me know!

**Comment by pbrubaker** [comment]
Excellent thanks Matt!  I’ll create a new issue when the next version is released!

**Comment by mattgodbolt** [comment]
Fab; I'll close this one off and await the new issue :)

**Comment by pbrubaker** [comment]
Cheers!
